### PR TITLE
disable browsing auto pan/scale in attribute table when showing visible features only

### LINKED ISSUE #34486: "automatically pan/zoom to the current feature" is broken
QGIS 3.10.2/master on Windows and Linux.

Open a attribute table in form view, then enable the "automatically pan to the current feature" or "automatically zoom to the current feature" and navigate the feature with the provided arrows.

No pan or zoom whatsoever.

Peek 2020-02-16 19-46

### PR DESCRIPTION
**Pull request by 3nids**
fixes #34486

### CODE REVIEW

File: src/gui/attributetable/qgsdualview.cpp [hunk 1]
```diff
@@ -486,6 +501,22 @@ void QgsDualView::panOrZoomToFeature( const QgsFeatureIds &featureset )
...
+    return;
+
+  mBrowsingAutoPanScaleAllowed = allowed;
+
+  mAutoPanButton->setEnabled( allowed );
+  mAutoZoomButton->setEnabled( allowed );
+
+  QString disabledHint = tr( "(disabled when attribute table ony shows features visible in the current map canvas extent)" );
+
+  mAutoPanButton->setToolTip( tr( "Automatically pan to the current feature" ) + ( allowed ? QStringLiteral( "" ) : QStringLiteral( " " ) + disabledHint ) );
```
Comment (nyalldawson) [inline src/gui/attributetable/qgsdualview.cpp]:
```suggestion
  mAutoPanButton->setToolTip( tr( "Automatically pan to the current feature" ) + ( allowed ? QString() : QString( ' ' ) + disabledHint ) );
```

**Review by nyalldawson — COMMENTED** [review]
LGTM

File: src/gui/attributetable/qgsdualview.cpp [hunk 2]
```diff
@@ -486,6 +501,22 @@ void QgsDualView::panOrZoomToFeature( const QgsFeatureIds &featureset )
...
+
+  mBrowsingAutoPanScaleAllowed = allowed;
+
+  mAutoPanButton->setEnabled( allowed );
+  mAutoZoomButton->setEnabled( allowed );
+
+  QString disabledHint = tr( "(disabled when attribute table ony shows features visible in the current map canvas extent)" );
+
+  mAutoPanButton->setToolTip( tr( "Automatically pan to the current feature" ) + ( allowed ? QString() : QString( ' ' ) + disabledHint ) );
+  mAutoZoomButton->setToolTip( tr( "Automatically zoom to the current feature" ) + ( allowed ? QStringLiteral( "" ) : QStringLiteral( " " ) + disabledHint ) );
```
Comment (3nids) [inline src/gui/attributetable/qgsdualview.cpp]:
```suggestion
  mAutoZoomButton->setToolTip( tr( "Automatically zoom to the current feature" ) + ( allowed ? QString() : QString( ' ' ) + disabledHint ) );
```

**Comment by 3nids** [comment]
the push job has been killed, but PR job is green

**Pull request merged**